synthesizer/src/block/transactions/mod.rs

```diff
@@ -63,88 +63,125 @@ impl<'a, N: Network> FromIterator<&'a Transaction<N>> for Transactions<N> {
 
 impl<N: Network> Transactions<N> {
     /// Returns `true` if the transactions contains the given transition ID.
+    #[cfg(feature = "parallel")]
     pub fn contains_transition(&self, transition_id: &N::TransitionID) -> bool {
-        match cfg!(feature = "parallel") {
-            true => self.par_values().any(|tx| tx.contains_transition(transition_id)),
-            false => self.values().any(|tx| tx.contains_transition(transition_id)),
-        }
+        self.par_values().any(|tx| tx.contains_transition(transition_id))
+    }
+
+    #[cfg(not(feature = "parallel"))]
+    pub fn contains_transition(&self, transition_id: &N::TransitionID) -> bool {
+        self.values().any(|tx| tx.contains_transition(transition_id))
```

Comment:
Might be worth making a `cfg_values` macro like the existing `cfg_iter`, `cfg_iter_mut`, etc. macros